db/revision_cache_lru.go

```diff
@@ -100,31 +100,36 @@ type revCacheValue struct {
 	removed     bool
 }
 
+// NewBackingStoreMap will return a new empty map for mapping collection ID to its associated rev cache backing store
+func NewBackingStoreMap() map[uint32]RevisionCacheBackingStore {
+	return make(map[uint32]RevisionCacheBackingStore)
+}
+
```

Comment:
This pattern of ~empty constructors (for maps/slices) results in callers being unable to preallocate capacity. I think we should just remove it and make callers use `make` directly.